packages/client/modules/userDashboard/components/OrgBilling/OrgPage.tsx

```diff
@@ -39,7 +37,7 @@ const OrgPage = (props: Props) => {
   const match = useRouteMatch<{orgId: string}>('/me/organizations/:orgId')!
 
   return (
-    <Container>
+    <Container className={'px-[16px] md:px-[48px]'}>
```

Comment:
+1 curious why you'd like to add the padding here? This feels more tailwind-y, but if we want to go with tailwind in this component, I think we should remove the styled components and use a `div`.

The way we had the padding previously would be more consistent with the rest of the app.

Reply:
I've used Tailwind where we have changes for breakpoints because I think tailwind handles that very cleanly. I didn't want to move _everything_ to Tailwind and break the work that was done previously / spend too long recreating styles. LMK which you think will be better for now & in the long run

Reply:
I think using Tailwind but using it with a `div` instead of `Container` would be a good move, but it's more personal preference than anything

Reply:
also, as a side effect tailwind scale ie. p-1, p-2, p-8 etc gives use a nice and consistent grid for all the elements. after a while things start to look nice, as there are no random values ;)